git_machete/client.py

```diff
@@ -1363,8 +1363,13 @@ def sync_annotations_to_github_prs(self) -> None:
         if not url_for_remote:
             raise MacheteException(fmt('No remotes defined for this repository (see `git remote`)'))
 
-        optional_org_name_for_github_remote: Dict[str, Optional[Tuple[str, str]]] = {
-            remote: get_parsed_github_remote_url(url) for remote, url in url_for_remote.items()}
+        optional_org_name_for_github_remote: Dict[str, Optional[Tuple[str, str]]] = {}
```

Comment:
While you're here... I'd replace tuple here with some simple class with 2 named fields

Reply:
Yeah, was thinking the same but i just wanted to use namedTuple instead of class, wdyt?

Reply:
Oh! `NamedTuple`, I forgot about it &mdash; yeah, sounds like a perfect choice here